### PR TITLE
MGDCTRS-1869 MGDCTRS-1870 Add missing config options to connector catalog

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/MGDCTRS-1869
https://issues.redhat.com/browse/MGDCTRS-1870

### CODE REVIEW

**Comment by Naros** [comment]
Hi @ani-sha, I added a comment to MGDCTRS-1870.  Ideally, we need an upstream change to the schema generation tooling to specify which `debezium-storage-xxxx` module's configuration needs to be loaded and appended to each of the connector's configuration definitions to return to what we had with Debezium 1.x.

**Review by Naros — APPROVED** [review]
Hi @ani-sha, LGTM.

**Comment by rk3rn3r** [comment]
@ani-sha @Naros You've missed calling the `./gen_templates.sh` script that updates `./templates/cos-fleet-catalog-debezium.yaml`. I will do this now.

**Comment by rk3rn3r** [comment]
Was this a manual edit? Or did you update the whitelist and triggered a schema generator build?
I'll look into this.

File: templates/cos-fleet-catalog-debezium.yaml [hunk 1]
```diff
@@ -459,7 +473,7 @@ objects:
           "icon_href" : "http://example.com/images/debezium-postgres-2.0.1.Final.png",
           "schema" : {
             "title" : "Debezium PostgreSQL Connector",
-            "required" : [ "topic.prefix", "database.hostname", "database.user", "database.dbname" ],
+            "required" : [ "topic.prefix", "database.hostname", "database.user" ],
```
Comment (rk3rn3r) [inline templates/cos-fleet-catalog-debezium.yaml:476]: @Naros is `database.names` really an optional config option?

**Pull request merged**